nautobot_chatops/dispatchers/base.py

```diff
@@ -44,8 +44,8 @@ def subclasses(cls):
         if settings.PLUGINS_CONFIG["nautobot_chatops"].get("ms_teams"):
             from .ms_teams import MSTeamsDispatcher
 
-        if settings.PLUGINS_CONFIG["nautobot_chatops"].get("webex_teams"):
-            from .webex_teams import WebExTeamsDispatcher
+        if settings.PLUGINS_CONFIG["nautobot_chatops"].get("webex"):
+            from .webex import WebExDispatcher
```

Comment:
How has this ever worked?? :-)

Need to change these to `"enable_ms_teams"` and `"enable_webex"` respectively.

Reply:
This entire `subclasses` function doesn't appear to be called at all during normal ChatOps interactions.  I would suggest holding off on changing these for now, and rewriting them in a separate PR (or remove them completely if found to not be needed).
@glennmatthews What do you think?

Reply:
Looks like it's mainly used by the `AccessLookupView` API view class, which is used in in the AccessGrant create/edit views to facilitate looking up the user/channel ID based on the user/channel name. They're clearly wrong as implemented so I'd prefer to fix these as part of this PR as long as you're touching this code anyways.

Reply:
Updated/fixed